### PR TITLE
bump version to 8.4.2

### PR DESCRIPTION
**Pull request by simitt**
## Motivation/summary

Bump version to 8.4.2

### CODE REVIEW

File: apmpackage/apm/changelog.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # newer versions go on top
 #
 # change type can be one of: enhancement, bugfix, breaking-change
-- version: "8.4.1"
+- version: "8.4.2"
```
Comment (kruskall) [inline apmpackage/apm/changelog.yml:4]: **question**: shouldn't this go on top of `8.4.1` ?

**Review by kruskall — APPROVED** [review]
LGTM! Just a minor question

File: apmpackage/apm/changelog.yml (hunk 1 above)
Comment (simitt) [reply apmpackage/apm/changelog.yml:4]: We haven't actually had any changes for the `8.4.1` package, so we are not releasing it and only bump the version to `8.4.2`.

Comment (kruskall) [reply apmpackage/apm/changelog.yml:4]: Ah, that entry is a placeholder 🤦

**Pull request merged**